velox/vector/tests/VectorTest.cpp

```diff
@@ -2495,3 +2495,19 @@ TEST_F(VectorTest, resizeArrayAndMapResetOffsets) {
     checkIndices(sizes);
   }
 }
+
+TEST_F(VectorTest, getRawStringBufferWithSpace) {
```

Comment:
Would you also add a test for the case when there is no existing buffer with sufficient space?